account_tax_balance/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Lorenzo Battistini - Agile Business Group
+# Copyright 2016 Tecnativa - Antonio Espinosa
+# Copyright 2016 ACSONE SA/NV - Stéphane Bidoul
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Tax Balance",
+    "summary": "Compute tax balances based on date range",
+    "version": "11.0.1.0.0",
+    "category": "Accounting & Finance",
+    "website": "https://www.agilebg.com/",
```

Comment:
Should we change as discussed in the list to the OCA webpage?

Reply:
Well, it can be, but it's not a priority